metadata-ingestion/src/datahub/ingestion/source/aws/sagemaker_processors/feature_groups.py

```diff
@@ -197,11 +200,10 @@ def get_feature_wu(
 
                 full_table_name = f"{glue_database}.{glue_table}"
 
-                self.report.report_warning(
-                    full_table_name,
-                    f"""Note: table {full_table_name} is an AWS Glue object.
+                logging.info(
+                    f"""Note: table {full_table_name} is an AWS Glue object. This source does not ingest all metadata for Glue tables.
```

Comment:
the `"""` with newlines will make the subsequent lines have weird indentation

use `textwrap.dedent` or do something like this

```
   logging.info(f"""\
Note...
second line
""")
```